clients/admin-ui/src/features/system/DescribeSystemStep.tsx

```diff
@@ -65,9 +67,10 @@ const DescribeSystemStep = ({
         : defaultInitialValues,
     [passedInSystem]
   );
-  const [createSystem] = useCreateSystemMutation();
-  const [updateSystem] = useUpdateSystemMutation();
-  const [isLoading, setIsLoading] = useState(false);
+  const [createSystemMutationTrigger, createSystemMutationResult] =
+    useCreateSystemMutation();
+  const [updateSystemMutationTrigger, updateSystemMutationResult] =
+    useUpdateSystemMutation();
```

Comment:
Yes, the naming scheme of `<operation><Model>Mutation<Trigger|Result>` is verbose. But the more I encounter code where the API-calling function starts off as a simple name, then gets wrapped in handling logic, and so on, the more I think making it clear which one actually fires the request is worth the characters.